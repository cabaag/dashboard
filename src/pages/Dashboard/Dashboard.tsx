import React from 'react';

// tslint:disable-next-line: variable-name
const Dashboard = () => {
	return (
		<div>Dashboard</div>
	);
};

export default Dashboard;
