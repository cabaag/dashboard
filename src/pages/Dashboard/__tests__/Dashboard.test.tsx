import { shallow } from 'enzyme';
import React from 'react';
import Dashboard from '../Dashboard';

describe('test Dashboard component', () => {

		test('should render component', () => {
				expect(shallow(<Dashboard />)).toMatchSnapshot();
		});

});
