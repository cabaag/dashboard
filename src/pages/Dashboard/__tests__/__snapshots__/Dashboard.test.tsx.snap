// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test Dashboard component should render component 1`] = `ShallowWrapper {}`;
